nest_elephant_tvb/translation/mpi_translator.py

```diff
@@ -6,12 +6,46 @@
 
 lock_status=Lock() # locker for manage the transfer of data from thread
 
-def init(path, param, comm_receiver, comm_sender, logger_master, logger_receive, logger_send):
+def init(path, param, comm, comm_receiver, comm_sender, loggers):
+    '''
+    Initialize the translation with MPI
```

Comment:
The comment does not add that much information. Maybe write something about what entails this initialization?

Reply:
```suggestion
    Initialize the translation with MPI. NOTE: more information will be added with more changes. Currently the multithreaded version is still in use.
```